nestjs/src/discord-servers/discord-servers.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { DiscordServer } from '@entities/discordServer';
+import { CreateDiscordServerDto, UpdateDiscordServerDto } from './dto';
+
+@Injectable()
+export class DiscordServersService {
+  constructor(
+    @InjectRepository(DiscordServer)
+    private repository: Repository<DiscordServer>,
+  ) {}
+
+  public async getById(id: number) {
+    return this.repository.findOne(id);
+  }
+
+  public async getAll() {
+    return this.repository.find();
+  }
+
+  public async create(data: CreateDiscordServerDto) {
+    const { id } = await this.repository.save(data);
+    return this.getById(id);
+  }
+
+  public async update(id: number, data: UpdateDiscordServerDto) {
+    await this.repository.update(id, data);
+    return this.getById(id);
```

Comment:
wondering, why to query by id here and in `create`?  both methods should return updated object

Reply:
You're right.  I've copied it from `disciplines`. Seems that we'll also need to fix that too, and check another ones..